table_helper.cc

```diff
@@ -129,31 +129,54 @@ future<> table_helper::setup_keyspace(cql3::query_processor& qp, service::migrat
         co_return;
     }
 
-    // FIXME: call `announce` once (collapse the calls here and in `setup_table`)
+    // FIXME: call `announce` once (`announce` keyspace and tables together)
```

Comment:
You had some nice suggestions on how to change things more radically (e.g., drop this table_helper) in the commit message. It's a shame nobody will see it later. Maybe you should copy some of them into this comment.

Reply:
Sure.

Can I consider this suggestion to be a general way to go in this kind of situation? I saw FIXMEs with a one-line comment and a lot of information (like suggestions on how to fix them) in the commit message, so I'm a bit confused.

Reply:
Sometimes the way to fix something is fairly obvious, we just need to remember to do it, so the FIXME doesn't need to be long. Here what bothered me wasn't the shortness of the FIXME per se, but rather the fact that the FIXME suggests we should do one thing (call announce once) whereas the commit message suggests a completely different approach (removing this function altogether, not fixing it). So you should decide which fix you're proposing, and if it's the one in the commit message - the FIXME shouldn't be this line you added here but a FIXME in another place saying "don't use table_helper, use X and Y instead".